src/types/block_data.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include "block_body.hpp"
 #include "block_header.hpp"
-#include "block_signature.hpp"
+#include "block_signatures.hpp"
```

Comment:
```suggestion
#include "types/block_signatures.hpp"
```
and for other inclusions